src/components/claiming/WalletSeedBackup.vue

```diff
@@ -5,7 +5,7 @@
     title="Wallet seed phrase backup"
     previousText="Back"
     nextText="Continue"
-    :previousStep="() => this.$router.push('/claiming/disclaimer')"
+    :previousStep="() => this.$router.push(`/claiming/disclaimer/${5}`)"
```

Comment:
This can be disclaimers length agnostic
```suggestion
    :previousStep="() => this.$router.push(`/claiming/disclaimer/${this.disclaimers.length - 1}`)"
```